utils/Constants.ts

```diff
@@ -1,60 +1,108 @@
+export const wrappedSecondaryTokenData: { [key: string]: ERC20TokenData } = {
+  mainnetFork: { symbol: "WBTC", address: "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599", decimals: 8 },
+  rsk: { symbol: "ETHs", address: "0x1D931Bf8656d795E50eF6D639562C5bD8Ac2B78f", decimals: 18 }, // Since RBTC is the native token on RSK
+  arbitrumTestnet: { symbol: "WBTC", address: "0x1F7dC0B961950c69584d0F9cE290A918124d32CD", decimals: 8 }
 };
 
-export const wBTC_address: { [key: string]: string } = {
-  mainnetFork: "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
-  rsk: "0x1D931Bf8656d795E50eF6D639562C5bD8Ac2B78f" // actually eths, reversed to reflect rbtc native coin
-};
-
-export const wBTC_precision: { [key: string]: number } = {
-  mainnetFork: 8,
-  rsk: 18 // actually eths, reversed to reflect rbtc native coin
-};
-
-export const EUR_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0xb49f677943BC038e9857d61E7d053CaA2C1734C1"
-};
-
-export const XAU_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0x214ed9da11d2fbe465a6fc601a91e62ebec1a0d6"
-};
-
-export const GBP_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0x5c0ab2d9b5a7ed9f470386e82bb36a3613cdd4b5"
-};
-
-export const ETH_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
-};
-
-// we only ue this feed in tests moce it out
-export const BTC_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c"
-};
-
-export const BTC_ETH_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0xdeb288F737066589598e9214E782fa5A8eD689e8"
+// TODO: Do we really need it when Blindex doesn't host the DEX?
+export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: string; decimals: number } } = {
+  mainnetFork: { symbol: "USDC", address: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", decimals: 6 },
+  rsk: { symbol: "XUSD", address: "0xb5999795BE0EbB5bAb23144AA5FD6A02D080299F", decimals: 18 }
 };
 
+// TODO: Do we really need it?
 export const NATIVE_TOKEN_NAME: { [key: string]: string } = {
   mainnetFork: "ETH",
-  rsk: "RBTC"
+  rsk: "RBTC",
+  arbitrumTestnet: "ETH"
 };
 
+// TODO: Do we really need it?
 export const SECONDARY_COLLATERAL_TOKEN_NAME: { [key: string]: string } = {
   mainnetFork: "BTC",
-  rsk: "ETHs"
+  rsk: "ETHs",
+  arbitrumTestnet: "WBTC"
+};
+
+// TODO: Maybe combine it with the price feeds we deploy? That way we could have a field called "shouldUpdate" and the BE bot could read it to know if it should update it or not. That way we won't need it hardcoded on the BE config like it is right now (in the updaterSettings section in the BE environment file).
+export const chainlinkPriceFeeds: { [key: string]: ChainlinkPriceFeed } = {
+  EUR_USD_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0xb49f677943BC038e9857d61E7d053CaA2C1734C1"
+    },
+    arbitrumTestnet: null,
```

Comment:
This is what I initially did.
I then changed it to this approach so it will be clearer that we said there is no price feed there.

I thought it will encourage us to always look if this price feed was added in the meanwhile.

WDYT?

Reply:
I don't think it's really necessary and cleaner will be better.. but it's NIT so you can decide whatever you think